.ci/docker/common/install_onnx.sh

```diff
@@ -23,3 +23,18 @@ pip_install \
 
 # TODO: change this when onnx-script is on testPypi
 pip_install "onnx-script@git+https://github.com/microsoft/onnx-script@29241e15f5182be1384f1cf6ba203d7e2e125196"
+
+IMPORT_SCRIPT_FILENAME="/tmp/onnx_import_script.py"
+# Cache the transfomers model to be used later by ONNX tests. By default, the model
+# is cached at ~/.cache/huggingface/hub/
+as_jenkins echo 'import transformers; transformers.AutoModel.from_pretrained("sshleifer/tiny-gpt2");' > "${IMPORT_SCRIPT_FILENAME}"
```

Comment:
Where does the transformers package get installed? I was initially  surprised since pytorch is not yet installed, and later realized it doesn't get invoked until later

Does transformers get installed as part of the torch install? If so, then consider moving this step below the install for a more understandable ordering

Reply:
The `transformers` package is installed just before this at https://github.com/pytorch/pytorch/blob/master/.ci/docker/common/install_onnx.sh#L22.  It doesn't need PyTorch (optional), but that specific `transformers.AutoModel.from_pretrained` function asks for either PyTorch, Tensorflow, or another framework that I couldn't remember before it starts downloading the model (may be to choose the correct format for PyTorch).  So I just pull in nightly to satisfy it.